docs/dataset.md

```diff
@@ -5,15 +5,15 @@ Promscale stores some configuration information in the Postgres database which i
 Setup is done by using the `-startup.dataset.config` flag which should contain this configuration structure in YAML format
 
 Example usage:
-```
+```bash
 promscale -startup.dataset.config=$'metrics:\n  default_chunk_interval: 6h'
 ```
 
-Expected format is YAML and currently only option that can be set is default chunk interval.
+The expected format is YAML.
 
-Example configuration in config.yaml:
+The YAML values are easier to set when using `config.yaml` instead of the cli flags. This would look as follows:
 
-```
+```yaml
 startup.dataset.config: |
```

Comment:
Is this line needed in the YAML file?

Reply:
it's not needed at all, but it tells the markdown syntax highlighter to highlight as yaml.

Reply:
Sorry, I meant line 17: `startup.dataset.config: |`

Reply:
I'm not sure that I understand exactly what your question is (I think it has to do with the pipe `|`, but I'm not sure).

Basically all cli flags can be specified in `config.yaml`, and then passed to promscale with `promscale -config config.yaml`. In this case, the `startup.dataset.config` parameter must itself be a yaml-formatted string. The only way to do this is with the `|`.

Maybe you mean, is that config param mandatory? No.